### PR TITLE
Create backup postgres github action

### PR DESCRIPTION
**Pull request by RMcVelia**
## Context
Create a reusable action to backup a postgres database

## Changes proposed in this pull request
Add backup-postgres action

## Guidance to review
https://github.com/DFE-Digital/register-trainee-teachers/actions/runs/9644386638  

## Checklist

- [ ] I have performed a self-review of my code, including formatting and typos
- [ ] I have [cleaned the commit history](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)
- [ ] I have added the `Devops` label
- [ ] I have attached the pull request to the trello card

### CODE REVIEW

**Comment by RMcVelia** [comment]
[DR - create offline postgres backup github action](https://trello.com/c/hHCNh3iR/1857-dr-create-offline-postgres-backup-github-action)

File: backup-postgres/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Restore Postgres Backup
```
Comment (saliceti) [inline backup-postgres/README.md:1]: The description should be updated

File: backup-postgres/action.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      uses: DFE-Digital/github-actions/set-kubectl@master
+
+    - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
+      with:
+        azure-credentials: ${{ inputs.azure-credentials }}
+
+    - name: K8 setup
+      shell: bash
+      run: |
+        make ci ${{ inputs.environment }} get-cluster-credentials
```
Comment (saliceti) [inline backup-postgres/action.yml:48]: As discussed, we should remove the dependency on make commands

File: backup-postgres/README.md (hunk 1 above)
Comment (RMcVelia) [reply backup-postgres/README.md]: fixed

File: backup-postgres/action.yml (hunk 2 above)
Comment (RMcVelia) [reply backup-postgres/action.yml]: removed dependency on make

**Pull request merged**